### PR TITLE
port forward range test: fix an oops

### PR DESCRIPTION
**Pull request by edsantiago**
Wrong variable.

Fixes: #14854

Signed-off-by: Ed Santiago <santiago@redhat.com>
```release-note
None
```

### CODE REVIEW

File: test/system/500-networking.bats [hunk 1]
```diff
@@ -677,8 +677,8 @@ EOF
 @test "podman run port forward range" {
     for netmode in bridge slirp4netns:port_handler=slirp4netns slirp4netns:port_handler=rootlesskit; do
         local range=$(random_free_port_range 3)
-        local port="${test%-*}"
-        local end_port="${test#-*}"
+        local port="${range%-*}"
+        local end_port="${range#-*}"
```
Comment (edsantiago) [inline test/system/500-networking.bats:681]: @Luap99 PTAL. The obvious question is, how did this ever work?

Comment (Luap99) [reply test/system/500-networking.bats:681]: It simply did not work. We just never looped in `for port`.

But this is not even the actual problem I was reporting. The test picked `5354-5356` which is wrong, 5355 is always blocked. So the free port logic has a bug as well.

**Comment by edsantiago** [comment]
Testing this on my laptop, rootless, I'm reliably getting:
```console
$ bin/podman run --network bridge -p 5933-5955:5933-5955 -d quay.io/libpod/testimage:20220615 top
Error: failed to mount runtime directory for rootless netns: no such file or directory
```
Looks like #13703, but this is f36, and it's not an SELinux issue (fails with `setenforce 0`). This morning's `dnf upgrade` brought in a new `glibc`, could that be it?

**Comment by edsantiago** [comment]
Thanks, that fixed it. I wonder if the error message could be fixed so it offers that command (rm -f ...) as a suggestion? That will be helpful to people who get that error message without having a clue what happened or how to fix it. Like me.

**Comment by Luap99** [comment]
@edsantiago I think we also need:
```
diff --git a/test/system/helpers.bash b/test/system/helpers.bash
index 273e8d2f5..e4d44c4a9 100644
--- a/test/system/helpers.bash
+++ b/test/system/helpers.bash
@@ -300,7 +300,7 @@ function random_free_port_range() {
     while [[ $maxtries -gt 0 ]]; do
         local firstport=$(random_free_port)
         local all_ports_free=1
-        for i in $(seq 2 $size); do
+        for i in $(seq 1 $(($size - 1))); do
             if ! port_is_free $((firstport + $i)); then
                 all_ports_free=
                 break
```

right now it tests `port`, `port + 2`, `port + 3` so it failed to check `port + 1`

File: test/system/500-networking.bats [hunk 2]
```diff
@@ -677,16 +677,20 @@ EOF
 @test "podman run port forward range" {
     for netmode in bridge slirp4netns:port_handler=slirp4netns slirp4netns:port_handler=rootlesskit; do
         local range=$(random_free_port_range 3)
-        local port="${test%-*}"
-        local end_port="${test#-*}"
```
Comment (edsantiago) [inline test/system/500-networking.bats:681]: SUBTLE BUT IMPORTANT! Note the change of dash-asterisk to asterisk-star in this one!

**Pull request merged**